test/test-cases/additional-api-sync-mode-sqlite-test-cases.js

```diff
@@ -581,9 +581,11 @@ module.exports = (
 
       assert.isObject(res.body)
       assert.propertyVal(res.body, 'id', 5)
-      assert.isArray(res.body.result)
+      assert.isObject(res.body.result)
+      assert.isArray(res.body.result.res)
+      assert.isBoolean(res.body.result.nextPage)
 
-      const resItem = res.body.result[0]
+      const resItem = res.body.result.res[0]
```

Comment:
Is this a fix not related to the PR?

Reply:
`Fix getWeightedAveragesReport test case`: https://github.com/bitfinexcom/bfx-reports-framework/pull/268/commits/44cf79b26897106284dc22378972a4ef6dabc9bc
yes, it's stuck from the previous PR
this is not a significant fix, decided not to separate it into a separate PR